classes/02class/exercises/c02-network12-challenge/luizfds/c02-network12.txt

```diff
@@ -0,0 +1,26 @@
+- explain how was the process to create a VPC peering
+I teamed up with Christina to be able to finish this exercise. She configured and sent to me a VPC peering invitation that and then I accepted.
+
+- add routes from origin VPC to target VPC - only for private subnets
+I added a new record at my route table `devopsacademy-rt-public` with Christina EC2 instance IP `192.168.0.0/16` and the target was our vpc peering `pcx-06fe6ee73bb4bdca1`. She did the same at her end to be able to connect with my VPC.
+
+- open any network ACL to allow the connection within the subnets -- try to only open the mininum required
+Christina did this configuration at her end allowing my private IP `10.0.1.251` to access her VPC.
+
+- from an instance in one vpc, SSH to the other instance (will have to share the ssh keys, create a new one just for this exercise -- pay attention to security)
+We have share our ssh keys using the command `cat .ssh/authorized_keys`.
+To add ssh keys to my machine I used the command `echo public_key_string >> ~/.ssh/authorized_keys`
+n the above command, substitute the public_key_string with the output from the `cat .ssh/authorized_keys`
+
+After given access to inbound traffic in security group we were able to connect in each other EC2 instance using ssh. See log from my EC2 below:
+
+ec2-user@ip-10-0-1-251 ~]$ echo ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCrkaRZD3oYFkZq1UrwokqX7XZybbVq3dDQDuNxO0c4i4j6MRXfsqHVdAP6ADpPzZjMuT+Y+2s051jdtI8Nj5sq8GfHQW6La5PfLss7FqLcquxbQvZuSgoUBBGPvJ72XgAg1HdMBFO9y6l62zhoBWEUW+s/l//POvTly+MBGCSpSsnmRAh3FX48l6PSISqwIgeZ5kSNL/1j6tHD15SHIIXYMb+WhYE+2C/Tx+pbOqRWcFxSSMSVxZnXTWuw1M8xorxwKCL14TGnRfD1E+KwbBjPxcl6WUBX71Q9Kc9dZWbgK8e1t2DhSSOjaLcCfqtn6zfTMzb1WzJDEOSI4lEYGeyN Christina AWS >> ~/.ssh/authorized_keys
```

Comment:
no need to log the public key here, you can remove it.. and make sure to also delete this key from your account now that is exposed on git

Reply:
Sure, I will do that. The EC2 used in this exercise is not running and it will be deleted.